be/src/exec/vectorized/csv_scanner.cpp

```diff
@@ -169,12 +200,24 @@ StatusOr<ChunkPtr> CSVScanner::get_next() {
                 CSVReader::Record dummy;
                 RETURN_IF_ERROR(_curr_reader->next_record(&dummy));
             }
+
+            if (_parse_options.skip_header) {
```

Comment:
If a user specify the skip_header as -1, will it hang?
Is there any restriction for the parameter when parsing? such as 0 <= skip_header <= i64.MAX ?